test/test_runner.sh

```diff
@@ -1,8 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
+TEST_USERNS=${TEST_USERNS:-}
+
 cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
+if test -n "$TEST_USERNS"; then
```

Comment:
``if [[ -n "$TEST_USERNS" ]]; then``

Reply:
I use `test` for the opposite reason, as I want to avoid the shell builtin, since in some shells `[` doesn't support all that is supported by test.  In any case, it is just stylistic in this case as it won't make any real difference.  I'll amend this change in a new version

Reply:
Ahh okay, thanks for the explanation.  Ignore my comment then, since consistency is more valuable.